### DISCUSSION TITLE
What type do you use to store an object as property that does not belong to a model?

### DISCUSSION THREAD

**Discussion by andrewdang17** [mobxjs/mobx-state-tree#1891] [Q&A] [answered]
I'm confused by the [types documentation](https://mobx-state-tree.js.org/overview/types) since there's not really any examples. Say I have a `User` model and I want to add a `pictures` where the object would be `{ versions: { original: "", thumbnail: "" } }`, how would I define that on the model? It seems like I would create a Picture model but that seems overkill to me, especially if I end up with something even more nested.

**Comment by EmilTholin** [comment, marked as answer]
Hi @andrewdang17!

If the nested models are used just for the `User` you can inline them if you prefer:

```ts
import { types } from "mobx-state-tree";

const User = types.model({
  id: types.identifier,
  name: types.string,
  pictures: types.model({
    versions: types.model({
      original: types.string,
      thumbnail: types.string
    })
  })
});

const user = User.create({
  id: "1",
  name: "John Doe",
  pictures: {
    versions: {
      original: "http://placekitten.com/800/800",
      thumbnail: "http://placekitten.com/200/200"
    }
  }
});
```
